src/pages/gitLookup.svelte

```diff
@@ -12,7 +12,7 @@
 			  const data = await response.json()	 
 			  output = `login: ${data.login}, bio: ${data.bio}, repos url: ${data.repos_url}`;
 			  picture = data.avatar_url;
-			  $players[players.length].name = input;
+			//   setCurrentPlayer(data.login)
```

Comment:
is this still needed?

Reply:
No, I think the file can be removed now.